### PR TITLE
Workflow

### PR DESCRIPTION
**Pull request by Robert-octavo**
TI create a pull request to check the workflow - how it's working

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+      - uses: actions/checkout@v2
+      - name: Use NodeJS
+        uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - run: npm ci 
+      - run: npm run format:check
+      - run: npm test -- --coverage
+        env:
+          CI: true
```
Comment (robert-octavo-web) [inline .github/workflows/ci.yml:19]: Ok

File: README.md [hunk 2]
```diff
@@ -68,3 +68,5 @@ This section has moved here: [https://facebook.github.io/create-react-app/docs/d
 ### `npm run build` fails to minify
 
 This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+
+this repository is for testing
```
Comment (robert-octavo-web) [inline README.md:72]: Ok

File: .github/workflows/ci.yml (hunk 1 above)
Comment (Robert-octavo) [reply .github/workflows/ci.yml:19]: ok

Comment (robert-octavo-web) [inline .github/workflows/ci.yml:19]: kk

File: README.md (hunk 2 above)
Comment (robert-octavo-web) [inline README.md:72]: ;;

**Review by robert-octavo-web — APPROVED** [review]

**Pull request merged**